packages/dds/map/api-report/map.api.md

```diff
@@ -261,7 +261,7 @@ export class SharedDirectory extends SharedObject<ISharedDirectoryEvents> implem
     values(): IterableIterator<any>;
 }
 
-// @public
+// @public @deprecated
 export class SharedMap extends SharedObject<ISharedMapEvents> implements ISharedMap {
```

Comment:
Our agreed POR was to leave SharedMap public, but deprecated, to point customers looking at FF 1.0 samples to the new SharedTree.